src/Drivers/Decorator.php

```diff
@@ -391,7 +407,7 @@ public function purge($features = null): void
             Collection::wrap($features)
                 ->map($this->resolveFeature(...))
                 ->pipe(function ($features) {
-                    $this->driver->purge($features);
+                    $this->driver->purge($features->all());
```

Comment:
These methods support collections, but are typed to expect arrays. Fixing this while I notice it.